platform/core/src/extensions/ExtensionManager.js

```diff
@@ -286,8 +286,15 @@ export default class ExtensionManager {
 
   _initHangingProtocolsModule = (extensionModule, extensionId) => {
     const { HangingProtocolService } = this._servicesManager.services;
-    extensionModule.forEach(({ id, protocol }) => {
-      HangingProtocolService.addProtocol(id, protocol);
+    extensionModule.forEach(({ id, name, protocol }) => {
```

Comment:
I'm confused, our protocols don't have `name`, where is this coming from?

Reply:
Removed.